drivers/mtd/nand/raw/pl35x-nand-controller.c

```diff
@@ -795,6 +795,25 @@ static int pl35x_nfc_exec_op(struct nand_chip *chip,
 				      op, check_only);
 }
 
+static unsigned long of_get_memclk_freq(const struct device_node *np, const int timing_mode)
+{
+	struct property	*prop;
+	const __be32 *cur;
+	int memclk_frequency = 0;
+	u32 memclk_mode;
+
+	of_property_for_each_u32(np, "memclk-timing-frequency", prop, cur, memclk_mode) {
```

Comment:
I've only seen `of_property_for_each_u32` being called with 3 arguments and it doesn't seem to be a variadic function either. What am I missing?

Reply:
@DeborahOoi96 I know this code isn't relevant anymore but I'd still like to understand how this worked.